src/deadline/client/ui/resources/blender_example_bundle/template.yaml

```diff
@@ -24,10 +24,10 @@ parameterDefinitions:
     description: Choose the file format to render as.
     default: PNG
     allowedValues: [TGA, RAWTGA, JPEG, IRIS, IRIZ, PNG, HDR, TIFF, OPEN_EXR, OPEN_EXR_MULTILAYER, CINEON, DPX, DDS, JP2, WEBP]
-  - name: RezPackages
+  - name: CondaPackages
```

Comment:
We can include both here, so the job bundle would work with either a Rez queue environment or a Conda queue environment providing the software.